WeatherMonitor/ConfigData/JsonConfigHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+namespace WeatherMonitor.ConfigData;
+#nullable disable
+public abstract class JsonConfigHandler
+{
+    private const string FilePath = @"C:\Users\User\Desktop\FTS\WeatherService\WeatherMonitor\WeatherMonitor\ConfigData\data.json";
```

Comment:
It is much better to have this path on `appsettings.json` file. Also, it must not have the full path in your device, it must be relative, like `..\ConfigData\data.json`